openbb_platform/providers/intrinio/openbb_intrinio/models/equity_info.py

```diff
@@ -53,15 +58,29 @@ async def aextract_data(
         """Return the raw data from the Intrinio endpoint."""
         api_key = credentials.get("intrinio_api_key") if credentials else ""
         base_url = "https://api-v2.intrinio.com"
-        url = f"{base_url}/companies/{query.symbol}?api_key={api_key}"
 
-        return await get_data_one(url, **kwargs)
+        async def callback(response: ClientResponse, _: Any) -> dict:
```

Comment:
Have you considered using `openbb_intrinioutils.helpers.response_callback()` function instead of adding this new one?
Although I see you did some custom adjustment with the `response_data["symbol"] = response.url.parts[-1]` line.

Reply:
Apparently it works the same. I'll use it thanks